polkadot/cli/src/cli.rs

```diff
@@ -19,8 +19,15 @@
 use clap::Parser;
 use std::path::PathBuf;
 
-/// The version of the node. The passed-in version of the workers should match this.
-pub const NODE_VERSION: &'static str = env!("SUBSTRATE_CLI_IMPL_VERSION");
+/// The version of the node.
+///
+/// This is the version that is used for versioning this node binary.
+/// By default the `minor` version is bumped in every release. `Major` or `patch` releases are only
+/// expected in very rare cases.
+///
+/// The worker binaries associated to the node binary should ensure that they are using the same
+/// version as the main node that started them.
+pub const NODE_VERSION: &'static str = "1.1.0";
```

Comment:
Why not `env!("CARGO_PKG_VERSION")`?

Reply:
I mean, okay, we don't want to use the commit hash in node version, but why the node version should be independent from the crate version in the first place?